### PR TITLE
fix tests after Beaker::Platform refactoring

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

**Comment by bastelfreak** [comment]
Since this is a fix for https://github.com/voxpupuli/beaker/commit/9d85e89062fc794b879b09f9852f0fe3669c03ea, which was first released in 6.4.0, should we bump the minimal version in the gemspec to 6.4.0?

**Comment by evgeni** [comment]
> Since this is a fix for [voxpupuli/beaker@9d85e89](https://github.com/voxpupuli/beaker/commit/9d85e89062fc794b879b09f9852f0fe3669c03ea), which was first released in 6.4.0, should we bump the minimal version in the gemspec to 6.4.0?

No.

1. this is a test-setup only change, the code is still compatible with older beaker
2. the tests actually pass on 6.3.0 (I tested! one test fails: "Beaker::Docker with a working connection #dockerfile_for uses pacman on archlinux" and that's because of https://github.com/voxpupuli/beaker/commit/18e6e2292aeed5557c6a85e45ba86365ca43d6f6)

**Comment by evgeni** [comment]
I also don't think this should be labeled `bug` -- it's not a bug in the code.

File: spec/beaker/hypervisor/docker_spec.rb [hunk 1]
```diff
@@ -683,48 +682,48 @@ module Beaker
...
-                                   })
+          dockerfile = docker.send(:dockerfile_for, make_host('none', {
+                                                                'platform' => Beaker::Platform.new('archlinux-current-x86_64'),
+                                                                'image' => 'foobar',
+                                                              }))
 
           expect(dockerfile).to match(/pacman --sync --refresh --noconfirm archlinux-keyring/)
           expect(dockerfile).to match(/pacman --sync --refresh --noconfirm --sysupgrade/)
-          expect(dockerfile).to match(/pacman --sync --noconfirm curl ntp net-tools openssh/)
+          expect(dockerfile).to match(/pacman --sync --noconfirm curl net-tools openssh/)
```
Comment (evgeni) [inline spec/beaker/hypervisor/docker_spec.rb:726]: we could make this
```suggestion
          expect(dockerfile).to match(/pacman --sync --noconfirm curl/)
```
and then it'd pass on 6.3 AND 6.4

**Review by bastelfreak — APPROVED** [review]

**Review by ekohl — APPROVED** [review]

**Pull request merged**